src/Microsoft.AspNetCore.Mvc.Razor.Host/TagHelperChunkVisitor.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Chunks;
+using Microsoft.AspNetCore.Razor.CodeGenerators;
+using Microsoft.AspNetCore.Razor.CodeGenerators.Visitors;
+using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Host
```

Comment:
This could be in internal namespace couldn't it? If so, you can nuke all the doc comments. If you want to keep them let me know and I can look em over.